lib/CL/devices/common.c

```diff
@@ -125,6 +126,19 @@ llvm_codegen (char *output, unsigned device_i, cl_kernel kernel,
   pocl_cache_final_binary_path (final_binary_path, program, device_i, kernel,
                                 command, specialize);
 
+#ifdef _WIN32
+  // Avoid race condition of multiple processes accessing same files.
```

Comment:
This is a C file, pls follow the C style guide.

Reply:
Fixed.